### PR TITLE
[RN Mobile] Track unsupported blocks list

### PR DESCRIPTION
**Pull request by daniloercoli**
This PR is a part of the work required to fix https://github.com/wordpress-mobile/gutenberg-mobile/pull/1199 and it does send the list of unsupported blocks up to the Native bridge when a post is opened.  Previously only a flag to indicate the presence of unsupported blocks was sent up to the bridge.

To test: See https://github.com/wordpress-mobile/gutenberg-mobile/pull/1199

### CODE REVIEW

**Review by marecar3 — APPROVED** [review]
Working as described!
LGTM! 👍

**Comment by etoledom** [comment]
Please don't merge until the iOS side is ready, since this change will break the iOS demo app.
I'll make take care of the needed changes asap :)

File: packages/edit-post/src/editor.native.js [hunk 1]
```diff
@@ -135,10 +135,8 @@ class Editor extends Component {
 		if ( ! prevProps.isReady && this.props.isReady ) {
 			const blocks = this.props.getEditorBlocks();
 			const isUnsupportedBlock = ( { name } ) => name === getUnregisteredTypeHandlerName();
-			const unsupportedBlocks = blocks.filter( isUnsupportedBlock );
-			const hasUnsupportedBlocks = ! isEmpty( unsupportedBlocks );
-
-			RNReactNativeGutenbergBridge.editorDidMount( hasUnsupportedBlocks );
+			const unsupportedBlocksName = blocks.filter( isUnsupportedBlock ).map( ( block ) => block.attributes.originalName );
+			RNReactNativeGutenbergBridge.editorDidMount( unsupportedBlocksName );
```
Comment (etoledom) [inline packages/edit-post/src/editor.native.js]: Very small detail:
For consistency we could change the name from `unsupportedBlocksName` to `unsupportedBlockNames`

**Review by etoledom — APPROVED** [review]
iOS side ready on https://github.com/wordpress-mobile/gutenberg-mobile/pull/1199 👍

File: packages/edit-post/src/editor.native.js (hunk 1 above)
Comment (daniloercoli) [reply packages/edit-post/src/editor.native.js]: Right. Renamed!

**Pull request merged**